scripts/inference/convert_hf_to_onnx.py

```diff
@@ -138,9 +138,9 @@ def export_to_onnx(
         with torch.no_grad():
             orig_out = model(**sample_input)
 
-        import onnx  # type: ignore
-        import onnx.checker  # type: ignore
-        import onnxruntime as ort  # type: ignore
+        import onnx
```

Comment:
for my own education, how did you get this to pass without these ignores?

Reply:
I didn't need to fix anything -- seems like the type ignores were superfluos, since `onnx` is part of the default install?

Reply:
interesting. when i wrote this i was getting type errors on those imports. oh well, glad it works now!